### PR TITLE
Add kubeadm function

### PR DESCRIPTION
**Pull request by velovec**

### CODE REVIEW

**Comment by dukov** [comment]
Can one of the admins verify this patch?

File: functions/kubeadm/namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```
Comment (dukov) [inline functions/kubeadm/namespace.yaml]: remove namespace

File: functions/kubeadm/cabpk-controller-manager.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/port: "8443"
+    prometheus.io/scheme: https
+    prometheus.io/scrape: "true"
+  labels:
+    control-plane: cabpk-controller-manager
+  name: cabpk-controller-manager-metrics-service
+  namespace: cabpk-system
```
Comment (dukov) [inline functions/kubeadm/cabpk-controller-manager.yaml]: remove NS

**Review by dukov — CHANGES_REQUESTED** [review]

File: functions/kubeadm/cabpk-controller-manager.yaml (hunk 2 above)
Comment (velovec) [reply functions/kubeadm/cabpk-controller-manager.yaml]: Removed

File: functions/kubeadm/namespace.yaml (hunk 1 above)
Comment (velovec) [reply functions/kubeadm/namespace.yaml]: Removed

**Pull request merged**